diff --git a/node_modules/react-native-ffmpeg/android/build.gradle b/node_modules/react-native-ffmpeg/android/build.gradle
index original..patched 100644
--- a/node_modules/react-native-ffmpeg/android/build.gradle
+++ b/node_modules/react-native-ffmpeg/android/build.gradle
@@ -14,7 +14,7 @@ buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:7.0.4'
@@ -24,7 +24,7 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 }